layouts/partials/collapsible-menu.html

```diff
@@ -21,38 +21,62 @@
 {{ end }}
 {{$currentSection := .Scratch.Get "currentSection" }}
 
+<!--- Sidebar layout --->
+{{ .Scratch.Set "sidebarLayout" "default" }}
+{{ if isset .Site.Params "sidebar_layout" }}
+  {{ .Scratch.Set "sidebarLayout" .Site.Params.sidebar_layout }}
+{{ end }}
+{{ if isset .Page.Params "sidebar_layout" }}
+  {{ .Scratch.Set "sidebarLayout" .Page.Params.sidebar_layout }}
+{{ end }}
+
+{{ $sidebarLayout := .Scratch.Get "sidebarLayout" }}
```

Comment:
this could be done through a trickle-down default statement (i.e. `.Page.Params.sidebar_layout | default .Site.Params.sidebar_layout | default "default"`) see head.html or footer.html layout code for other examples.)

Reply:
updated.